### PR TITLE
wifi: sap: Implement inactivity timer retrieval

### PR DESCRIPTION
**Pull request by krish2718**
This helps WPA supplicant kick off inactive STAs.

### CODE REVIEW

File: drivers/wifi/nrf700x/src/wpa_supp_if.c
Comment (sachinthegreen) [inline drivers/wifi/nrf700x/src/wpa_supp_if.c:2770]:
```suggestion
		LOG_ERR("%s: nrf_wifi_fmac_get_station failed", __func__);
```

**Review by sachinthegreen — APPROVED** [review]

**Comment by krish2718** [comment]
Waiting on @lave30  approval after tests

**Pull request merged**
